src/index.ts

```diff
@@ -215,11 +215,11 @@ export default class Account {
    * @param val
    * @param cb
    */
-  setStorage(trie: Trie, key: Buffer | string, val: Buffer | string, cb: () => void) {
+  setStorage(trie: Trie, key: Buffer | string, val: Buffer | string, cb: TriePutCb) {
```

Comment:
Interface is matching and also backwards-compatible, so this change should be ok.